ios/MeasureSDK/MeasureSDK/Config/DefaultConfig.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  DefaultConfig.swift
+//  MeasureSDK
+//
+//  Created by Adwin Ross on 25/08/24.
+//
+
+import Foundation
+
+public struct DefaultConfig {
```

Comment:
Does the default config need to be exposed as public API?

Reply:
Take a look into how we can print, validate the public API, looks like there is something in the [swift-package-manager](https://github.com/swiftlang/swift-package-manager/issues/4716).

This will help us in avoiding the issue discussed here: https://forums.swift.org/t/using-swift-api-digester/22956

Reply:
created a issue for it #1136